src/js/containers/crossFile/CrossFileContentContainer.jsx

```diff
@@ -42,6 +47,18 @@ class CrossFileContentContainer extends React.Component {
 		}
 	}
 
+	isGtas() {
+		ReviewHelper.isGtasWindow()
+			.then((res) => {
+				if(res.open != this.state.gtas) {
```

Comment:
Even in the previous setup, `res.open` would NEVER equal `this.state.gtas`. Given the suggestion below and the changes to what we return from the backend, this check could be removed completely and you could always just set state.